client/components/Home/AllStories.tsx

```diff
@@ -0,0 +1,39 @@
+import { View, Text, StyleSheet, Dimensions, Pressable, ScrollView, SafeAreaView } from 'react-native'
+import { useEffect } from 'react';
+import { useRouter } from 'expo-router';
+import { getSelectedProfile } from '@/services/apiService';
+import { useDataContext } from '@/context/globalContext';
+import { StoryType } from '@/types';
+import StoryList from '../Stories/StoryList';
+
+
+
+const AllStories = () => {
+
+  const router = useRouter()
+
+  const { setSelectedStory, setSelectedProfile, selectedProfile, selectedStory } = useDataContext();
+
+  useEffect(() => {
+    const fetchProfile = async () => {
+      if (selectedProfile) {
+        const updatedProfile = await getSelectedProfile(selectedProfile.id);
+        setSelectedProfile(updatedProfile);
+      }
+    }
+    fetchProfile();
+  }, [selectedStory, selectedProfile?.id]);
+
+  function handlePress(story: StoryType) {

```

Comment:
this can be moved to story list/item as its applicable to homeStories and favStories